snuba/query/query.py

```diff
@@ -151,9 +263,15 @@ def get_arrayjoin(self) -> Optional[str]:
     def get_having(self) -> Sequence[Condition]:
         return self.__body.get("having", [])
 
+    def get_having_exp(self) -> Optional[Expression]:
+        return self.__having
+
     def get_orderby(self) -> Optional[Sequence[Any]]:
         return self.__body.get("orderby")
 
+    def get_orderby_exp(self) -> Sequence[OrderBy]:
```

Comment:
`_exp` doesn't make a lot of sense here (since order by is not an expression.) I'm not exactly sure what to call these though — they're not clauses, either. 😕

Reply:
To me these methods will be renamed as get_orderby, get_conditions, etc. as soon as the AST is in prod and the old system based on the dictionary is  removed.